lib/gemstash/cli/start.rb

```diff
@@ -36,6 +37,12 @@ def store_daemonized
         Gemstash::Env.daemonized = daemonize?
       end
 
+      def check_rubygems_version
+        STDERR.puts("Rubygems version is too old, please update rubygems by running: " \
+                    "gem update --system") unless
+                    Gem::Requirement.new("~> 2.4").satisfied_by?(Gem::Version.new(Gem::VERSION))
```

Comment:
How about gem requirement of ">= 2.4" instead? Otherwise this will give an erroneous `too old` message if they go to 3.0.